src/components/base/FormattedVal/index.js

```diff
@@ -98,11 +98,13 @@ export function FormattedVal(props: Props) {
     if (withIcon && isNegative) {
       val *= -1
     }
+    const locale = settings ? settings.language : ''
```

Comment:
is there a risk of `settings` not being populated here ? I see it comes directly from `state.settings` that has at least a default state

Reply:
no, settings shouldn't be `?: settings` I guess it's part of the flow refactoring we need to do

Reply:
there is no risk, but in this file and some other files we shouldn't do something like 
settings?: Settings, in Props